### DISCUSSION TITLE
Problem with the video comments

### DISCUSSION THREAD

**Discussion by H4ck3r-Pr0** [TeamNewPipe/NewPipe#11023] [Questions] [answered]
When I watch any video, the comments don't charge. After, an application message says: "I'm sorry, something went wrong".
Is it a code error or is it mine?

**Comment by wb9688** [comment, marked as answer]
Please update to NewPipe v0.27.0 and it should work again.

**Comment by H4ck3r-Pr0** [comment]
The application had installed to the last version v0.27.0 and the proble is the same.

Firmado, Anonymous.

-------- Mensaje original --------
El 30/4/24 21:53, wb9688  ha escrito:

**Reply by wb9688** [reply]
Can you please share the crash log then?

**Comment by H4ck3r-Pr0** [comment]
The problem had been fixed (the update was downloaded to my phone but not
installed). Thanks for the support and sorry for the inconvenience.
I'm so sorry. :'(
